### ISSUE TITLE
Distorted scaled in 2-dim mesh plot

### ISSUE THREAD

**Issue by cbtxs** [weihuayi/fealpy#408]
Hello developer of FEALPy:
       I encountered a small issue while using FEALPy, when I try to plot a TriangleMesh, the aspect ratio of image is incorrect. For example, in this code:
```python
import numpy as np
import matplotlib.pyplot as plt
from fealpy.mesh.triangle_mesh import TriangleMesh

mesh = TriangleMesh.from_box([0, 4, 0, 1], nx=20, ny=5)
fig, axes = plt.subplots()
mesh.add_plot(axes)
plt.show()
```
my domain of mesh is $(0, 4)\times(0, 1)$, but the figure drawed by FEALPy is a square region. I would appreciate it if you can help me to solve this problem. Thank you!
![Figure_1](https://github.com/weihuayi/fealpy/assets/52527392/01a881e8-e3f9-49bf-96de-1571efe63b1e)

**Comment by AlbertZyy** [comment]
I have fixed this problem #411 . Now the aspect defaults to 'equal', so meshes can show their original aspect ratio.

**Comment by AlbertZyy** [comment]
I've opened #412 .
Now, the code
```python
from fealpy.mesh import TriangleMesh

mesh = TriangleMesh.from_box([0, 4, 0, 1], nx=8, ny=5)
from matplotlib import pyplot as plt
fig = plt.figure()
axes = fig.add_subplot()
mesh.add_plot(axes)
mesh.find_node(axes, showindex=True)
```
will generate
![tttt](https://github.com/weihuayi/fealpy/assets/108785827/970c3ea1-c487-4cb1-99e5-740f57a2c714)
